pkg/generate/app/cmd/newapp.go

```diff
@@ -29,6 +29,13 @@ import (
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
+const GeneratedByNamespace = "openshift.io/generated-by"
+
+var (
+	GeneratedByNewApp   = map[string]string{GeneratedByNamespace: "OpenShiftNewApp"}
```

Comment:
Exporting maps is awkward. Can you make all these consts and have `setAnnotations` construct the maps?